packages/react-core/src/components/Tabs/Tab.tsx

```diff
@@ -102,10 +109,29 @@ const TabBase: React.FunctionComponent<TabProps> = ({
 
   return (
     <li
-      className={css(styles.tabsItem, eventKey === localActiveKey && styles.modifiers.current, childClassName)}
+      className={css(
+        styles.tabsItem,
+        eventKey === localActiveKey && styles.modifiers.current,
+        handleTabClose && styles.modifiers.action,
```

Comment:
If a `.pf-m-action` tab is `disabled` or `.pf-m-aria-disabled`, this `<li>` needs the class `.pf-m-disabled`. This turns the entire item (tab and close button) grey, instead of just the tab which you will see if you disable a tab in the PR preview.

Also the core examples show the close buttons as enabled when a tab is disabled, meaning it's possible to have a disabled tab with either an enabled or disabled close button. Maybe @mcarrano or @mmenestr can provide clarification if that's valid, and if we want to update our examples to show a best practice. Assuming it's valid, I think we'll need a way to handle the disabled state of the close button in the react component.

Reply:
That's a really good question, and I'm not sure what's correct here. @mmenestr do you have thoughts? Maybe it's best to keep this flexible to allow for different use cases. There might be situations where you'd want to remove a disabled tab. Depends why the tab is disabled.

Reply:
Hmm yeah that's a good question. I think I agree with you @mcarrano, there could be a use case for being able to dismiss a disabled tab, so we should probably keep that option.

Reply:
Added an additional prop to control `isDisabled` for the close button separately.